quadcontrol/mma.c

```diff
@@ -153,30 +115,17 @@ void mma_done(void)
 
 int mma_init(const quad_coeffs_t *coeffs)
 {
-	unsigned int i;
-	int cnt, err = 0;
+	int err = 0;
 
 	if ((err = mutexCreate(&mma_common.lock)) < 0) {
```

Comment:
I know that it is not a part of Your PR, but I think it can be fixed to be consistent with MISRA:
```suggestion
        err = mutexCreate(&mma_common.lock)
	if ((err < 0) {
```

Reply:
Good idea! Applied.